python/lint/action.yaml

```diff
@@ -26,18 +26,18 @@ runs:
         python-version: ${{ inputs.python-version }}
     - name: Run isort linter
       if: ${{ inputs.use-isort == 'true' }}
-      uses: jamescurtin/isort-action@f555d1d11ca0529654eb6d1f017f9e4a269ffa7f
+      uses: jamescurtin/isort-action@f14e57e1d457956c45a19c05a89cccdf087846e5 # 1.1.0
```

Comment:
Just a quick heads up, the middleware team has been experimenting with [Ruff](https://github.com/astral-sh/ruff). While we don't have concrete results to report just yet, it's highly likely that we'll be looking into its potential for our services.